.github/workflows/pr.yml

```diff
@@ -0,0 +1,25 @@
+name: PR
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    # - uses: actions/cache@v1
+    #   with:
+    #     path: ~/.gradle
+    #     key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+    #     restore-keys: |
+    #       ${{ runner.os }}-gradle-
+    - uses: eskatos/gradle-command-action@v1
```

Comment:
I think we want to use the repositiries Gradle wrapper (`./gradlew`) rather than the version of Gradle that this action uses.

Reply:
I believe we are ...

![image](https://user-images.githubusercontent.com/388652/68328136-8fb5e600-0083-11ea-9466-206df007a350.png)